app/controllers/bulk_ingest_controller.rb

```diff
@@ -133,22 +135,49 @@ def build_change_set(attrs)
       change_set
     end
 
-    def selected_files_param
-      return {} unless params.key?(:selected_files) && !params[:selected_files].empty?
-      params[:selected_files].to_unsafe_h
+    # Retrieve the selected_files parameter from the request
```

Comment:
Do you think we should extract this code to a module that could be shared with BaseResourceController?  This doesn't need to be addressed in this PR, but it would be good to create an issue if the code can be shared.

Reply:
I've created https://github.com/pulibrary/figgy/issues/2784 to track that.